xrviz/dashboard.py

```diff
@@ -81,8 +98,25 @@ def create_plot(self, *args):
             if self.var in list(sel_data.coords):  # When a var(coord) is plotted wrt itself
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+            c_low_val = self.kwargs['cmap lower limit']
+            c_upp_val = self.kwargs['cmap upper limit']
+
+            # It is better to set initial values as 0.1,0.9 rather than 0,1 i.e. (min, max)
+            # to get a color balanced graph
+            c_lim_lower = float(c_low_val) if c_low_val else sel_data.quantile(0.1)
+            c_lim_upper = float(c_upp_val) if c_upp_val else sel_data.quantile(0.9)
+
+            color_range = {sel_data.name: (c_lim_lower, c_lim_upper)}
+
+            if not c_low_val:  # if initial values are empty
```

Comment:
0? You called `float` above, which will error on black or incorrect input.

Reply:
I have handled this case. Thanks!